react-api-client/Makefile

```
@@ -0,0 +1,19 @@
+# opentrons react-api-client makefile
+
+# using bash instead of /bin/bash in SHELL prevents macOS optimizing away our PATH update
+# TODO(mc, 2021-02-12): this may be unnecessary by using `yarn run` instead
+SHELL := bash
+
+# standard targets
+#####################################################################
+
+.PHONY: all
+all: clean build
+
+.PHONY: clean
+clean:
+	tsc --build --clean && yarn shx rm -rf \"dist\"
```

`tsc --build --clean` is handled by the top-level Makefile, I don't think we should include it here. Also the quote escaping is copypasta from package.json

```suggestion
	yarn shx rm -rf dist
```